test/tasks/Regression.t.sol

```diff
@@ -16,7 +16,7 @@ contract RegressionTest is Test {
             "0x174dea7100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000001200000000000000000000000005e6432f18bc5d497b1ab2288a025fbf9d69e22210000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000024b40a817c0000000000000000000000000000000000000000000000000000000005f5e100000000000000000000000000000000000000000000000000000000000000000000000000000000007bd909970b0eedcf078de6aeff23ce571663b8aa0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000024b40a817c0000000000000000000000000000000000000000000000000000000005f5e10000000000000000000000000000000000000000000000000000000000";
         vm.createSelectFork("mainnet", 21724199);
         MultisigTask multisigTask = new GasConfigTemplate();
-        multisigTask.simulateRun(taskConfigFilePath);
+        multisigTask.simulateRun(taskConfigFilePath, "");
```

Comment:
We should just have an overload of `simulateRun` that does not take a bytes arg, and internally that method just calls `simulateRun(taskConfigFilePath, "")`. This would be cleaner and more clear than empty strings and `0x` everywhere